### ISSUE TITLE
[Bug] [Notification] mentiont user on not-watching-repo = no notification

### ISSUE THREAD

**Issue by 6543** [go-gitea/gitea#9233]
If a user is mentiont in a issue of a repo he is not watching he will get no notification

gitea version: master (23ef9ff516b5d5168e6c9e8f6aa6d24f93602aca)

should have nothing to do with permission since the instance-admin is also afected

**Comment by guillep2k** [comment]
Was that in master? The user needs to have read permissions on issues + e-mail notification enabled.

**Comment by 6543** [comment]
@guillep2k sory for the les sinfos ... working on the empty issue dashboard!

**Comment by 6543** [comment]
this bug is stil anoying and block work on gitea.com :(

**Comment by guillep2k** [comment]
Sorry, @6543, I can't reproduce. With current `master` (b6fa229dcfbe4ad64fe758ef92abaf7a80e6f700), I did the following:

* I've created a public repo with **user1** (`user1/publicrepo`).
* With **user1**, I've created an issue in that repo with the following description: `I mention @user2`.
* **user2**, who wasn't watching that repo, got a proper mail notification.
* With **user2**, I set to watch `user1/publicrepo`. Then I set it to _unwatch_ `user1/publicrepo` in order to created an explicit unwatch.
* With **user1**, I've created a new issue with the following description: `I mention @user2 again`.
* **user2**, who explicitly unwatched that repo, got a proper mail notification.

I didn't get anything in **user2**'s _dashboard_, however. Is that what your issue is about?

**Comment by 6543** [comment]
I'll look & test later :)

**Comment by 6543** [comment]
tested with newest version:
1. user1 create repo (or has one witch user2 not watching)
2. user1 create issue
3. user1 create comment "soething @user2"
4. user2 get NO notify

**Comment by 6543** [comment]
after #9535 is merged I'm going to look for this issue again ... :)

**Comment by 6543** [comment]
@guillep2k just noticed yor question - jes its about gezing notify on UI too

**Comment by 6543** [comment]
one of the next things I'll work on

**Comment by 6543** [comment]
I'll test it this still is the case ...

**Comment by 6543** [comment]
Its in current master still the case

**Comment by zeripath** [comment]
Sorry @6543 I missed that you had confirmed that it was still present.

**Comment by 6543** [comment]
well - if I had more time ... I could had looked at it

**Comment by bircni** [comment]
This appears to be resolved since Gitea 1.14.0. PR #14218 reworked mention handling so that mentioned users receive notifications (and emails) directly, independent of whether they watch the repository. In the current code, a mention pushes a notification with a specific ReceiverID, and CreateOrUpdateIssueNotifications sends directly to that receiver rather than only to watchers. Closing as resolved; please reopen if you can still reproduce on a recent release.
